src/Mention.js

```diff
@@ -6,9 +6,12 @@ const styled = defaultStyle({
   fontWeight: 'inherit',
 })
 
-const Mention = styled(({ display, style }) => (
-  <strong {...style}>{display}</strong>
-))
+class Mention extends React.Component {
```

Comment:
Why did you change that from a purely functional component to a regular one? I'd love it to stay functional when possible.

Reply:
Hi, thanks for taking the time to look at the code,

I did it some time ago, and my intention was to have it working.
I remember that it was breaking with `styled` trying to add a ref to a non functional component, which resulted in unwanted warnings. I'll double check that

Reply:
Upon investigation I had this warning: [#19](https://github.com/jfschwarz/substyle/issues/19), It is fixed in substyle 6.1.1 so I'll update substyle from 6.0.0 to 6.1.1 and revert to using a functional component

Reply:
@frontendphil I just committed the changes